NeXus/NXxpcs.nxdl.xml

```diff
@@ -71,137 +78,357 @@
+            expected (:math:`\pm` error).  The data should be in the same format as ``g2``.
+        </doc>
+        <attribute name="storage_mode" type="NX_CHAR">
+          <enumeration>
+            <item value="one_array"/> 
+            <item value="data_exchange_keys"/>
+            <item value="other"/>
+          </enumeration>
+        </attribute>
       </field>
       <field name="G2_unnormalized" type="NX_NUMBER" units="NX_ARBITRARY_UNITS" minOccurs="0" maxOccurs="1">
-        <doc>unnormalized intensity auto-correlation function</doc>
+        <doc>
+            unnormalized intensity auto-correlation function.
+
+            Specifically, ``g2`` without the denominator.  The data should be in the same format as ``g2``.
+
+        </doc>
+        <attribute name="storage_mode" type="NX_CHAR">
+          <enumeration>
+            <item value="one_array"/> 
+            <item value="data_exchange_keys"/>
+            <item value="other"/>
+          </enumeration>
+        </attribute>
       </field>
       <field name="delay_difference" type="NX_INT" units="NX_INT" minOccurs="0" maxOccurs="1">
         <doc>
-          delay_difference (also known as delay or lag) quantized difference so that the "step" between two consecutive
+          delay_difference (also known as delay or lag step)
+
+          This is quantized difference so that the "step" between two consecutive
           frames is one frame (or step = dt = 1 frame)
-          It is the delay time corresponding to the g2 correlation values.
-          The unit of delay_differences is NX_INT for units of frames (i.e. integers) preferred,
-          refer to NXdetector for conversion to time units
+
+          It is the "quantized" delay time corresponding to the ``g2`` values.
+
+          The unit of delay_differences is NX_INT for units of frames (i.e., integers) preferred,
+          refer to NXdetector for conversion to time units. <!-- TODO make links for NX_INT and NXdetector -->
         </doc>
+        <attribute name="storage_mode" type="NX_CHAR">
+          <enumeration>
+            <item value="one_array"/> 
+            <item value="data_exchange_keys"/>
+            <item value="other"/>
+          </enumeration>
+        </attribute>
       </field>
     </group>
 
     <group type="NXdata" name="twotime" minOccurs="0">
-      <!-- TODO: needs documentation -->
+    <doc>
+      The data (results) in this section are based on the two-time intensity correlation function derived from a time series of scattering images.
+    </doc>
       <field name="two_time_corr_func" type="NX_NUMBER" units="NX_ANY" minOccurs="0" maxOccurs="1">
         <doc>
-          two-time correlation see Fluerasu, Phys Rev E (2007), Eq 1 and Sutton, Optics Express (2003)
+          two-time correlation of speckle intensity for a given q-bin or roi (represented by the nth roi_map value) 
+
+          See Fluerasu, Phys Rev E (2007), Eq 1 and Sutton, Optics Express (2003) for an early
```

Comment:
First author here, is the name spelled correctly?

Reply:
copy and paste from paper below.   Looks right

Slow dynamics and aging in colloidal gels studied by x-ray photon correlation spectroscopy
Andrei Fluerasu,